packages/create-astro/package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@types/degit": "^2.8.3",
     "@types/prompts": "^2.0.14",
+    "chalk": "^5.0.1",
```

Comment:
Used to render characters by custom hex values for the gradient. This should only add 1-2 KB to the final install size, which was much better than other "gradient generators" on npm. We can also swap out kleur for chalk if we feel _really_ strongly